livekit-api/livekit/api/_service.py

```diff
@@ -2,7 +2,7 @@
 import aiohttp
```

Comment:
Can you add  from `__future__ import __annotations__` in the beginning of the file?
This allows to use Union syntax like `SIPGrants | None` on older python versions

Reply:
Should I rollback Python version?

Reply:
@theomonnom I tried to rollback the version, but it doesn't recognize the future import. So I'll keep Python 3.10 then.

Reply:
I think it should be `from __future__ import annotations` ?

Reply:
That works, thank you!